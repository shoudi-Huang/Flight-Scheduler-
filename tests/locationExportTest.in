locations
location export
location export /User/locationExport.csv
location export ExportEmptyTest.csv
location add Mumbai 19.08 72.741 0.4
location export Export1Test.csv extra bit ha ha
location import locations.csv
location add NewDelhi 28.527 77.0688988 -0.123
locations
location export ExportLocationTest.csv
exit
