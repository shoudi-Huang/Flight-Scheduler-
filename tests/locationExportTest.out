User: Locations (0):
(None)

User: Error writing file.

User: Error writing file.

User: Exported 0 locations.

User: Successfully added location Mumbai.

User: Exported 1 location.

User: Imported 51 locations.

User: Successfully added location NewDelhi.

User: Locations (53):
AbuDhabi, Adelaide, AliceSprings, Alta, Athens, Auckland, Beijing, Berlin, Bern, Bordeaux, Brisbane, Cairo, Cardiff, Chicago, CoffsHarbour, Dallas, Darwin, Dubai, Dubbo, GoldCoast, Hanoi, Hobart, Houston, Jakarta, Johannesburg, Lagos, Liverpool, London, Longyearbyen, LosAngeles, Luton, Madrid, Manchester, Moscow, Mumbai, NewDelhi, NewYork, Orange, Oslo, Paris, Perth, Rio, Rome, SanFrancisco, Stockholm, Sydney, Toulouse, Townsville, Tromso, Ufa, Utqiagvik, Vladivostok, Washington

User: Exported 53 locations.

User: Application closed.
